components/Badge/Badge.js

```diff
@@ -5,20 +5,23 @@ import styles from './Badge.css';
 
 Badge.propTypes = {
   svgComponent: PropTypes.element.isRequired,
+  children: PropTypes.element,
   label: PropTypes.string,
   labelClassName: PropTypes.string,
 };
 
 Badge.defaultProps = {
+  children: undefined,
   label: '',
   labelClassName: undefined,
 };
 
-function Badge({ svgComponent, label, labelClassName }) {
+function Badge({ children, svgComponent, label, labelClassName }) {
   return (
     <div className={styles.Badge}>
       <div className={styles.iconContainer}>{svgComponent}</div>
       <div className={classNames(styles.centerText, styles.label, labelClassName)}>{label}</div>
+      {children}
```

Comment:
This lends itself to a bit of a shaky API - consider what you've gotta handle when passing children and then label.

I also don't like the idea of needing to hijack `labelClassName` like you would have to do otherwise.

Let's drop label as a prop and give total control of the children styling to the user via render prop (children).

Can you make children accept a prop type of element _or_ string _or_ node?

Also, when you make those changes, you'll need to fix the few live Badge implementations. It should just be copy-pasting styles from the Badge component to the implementation.

Lastly, don't forget to update the Story for Badge.

Thanks for doing so much work with us! I've got you on the list to get a t-shirt, and if you don't have time to make these changes, don't sweat it. You've already done a lot for me. ❤️